### PR TITLE
make issues url point to reporting-bugs instructions

### PR DESCRIPTION
**Pull request by vitorgalvao**
If we use this link directly even in CLI messages, perhaps more people will use the templates, and save us all a bunch of time.

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The idea is for each Cask to encapsulate and automate the story of how a given a
 
 __Yes, yes, yes!__ Please fork/pull request to update Casks, add features and clean up documentation! Anything you can do to help out is very welcome.
 
-It's also [__pretty darn easy__ to create Casks](../CONTRIBUTING.md), so please build more of them for the software you use. And if homebrew-cask doesn't support the packaging format of your software, please [open an issue](https://github.com/caskroom/homebrew-cask/issues) and we can get it working together.
+It's also [__pretty darn easy__ to create Casks](../CONTRIBUTING.md), so please build more of them for the software you use. And if homebrew-cask doesn't support the packaging format of your software, please [open an issue](https://github.com/caskroom/homebrew-cask#reporting-bugs) and we can get it working together.
```
Comment (jawshooah) [inline doc/FAQ.md]: > if homebrew-cask doesn't support the packaging format of your software

This seems more like a feature request to me.

Comment (vitorgalvao) [inline doc/FAQ.md]: Agreed. Changed.

File: doc/FAQ.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The idea is for each Cask to encapsulate and automate the story of how a given a
 
 __Yes, yes, yes!__ Please fork/pull request to update Casks, add features and clean up documentation! Anything you can do to help out is very welcome.
 
-It's also [__pretty darn easy__ to create Casks](../CONTRIBUTING.md), so please build more of them for the software you use. And if homebrew-cask doesn't support the packaging format of your software, please [open an issue](https://github.com/caskroom/homebrew-cask/issues) and we can get it working together.
+It's also [__pretty darn easy__ to create Casks](../CONTRIBUTING.md), so please build more of them for the software you use. And if homebrew-cask doesn't support the packaging format of your software, please [open an issue](https://github.com/caskroom/homebrew-cask/issues/new?title=Feature%20request%3A&body=%23%23%23%20Description%20of%20feature%2Fenhancement%0A%0A%0A%0A%23%23%23%20Justification%0A%0A%0A%0A%23%23%23%20Example%20use%20case%0A%0A%0A%0A) and we can get it working together.
```
Comment (jawshooah) [inline doc/FAQ.md]: In order to keep the template link up-to-date, this needs to be changed to `[open an issue][feature_request_template]`, and at the bottom of the document add

```
[feature_request_template]: https://github.com/caskroom/homebrew-cask/issues/new?title=Feature%20request%3A&body=%23%23%23%20Description%20of%20feature%2Fenhancement%0A%0A%0A%0A%23%23%23%20Justification%0A%0A%0A%0A%23%23%23%20Example%20use%20case%0A%0A%0A%0A
```

Then in `developer/bin/update_issue_template_urls`,  add `doc/FAQ.md` to `files_to_update`.

Comment (vitorgalvao) [inline doc/FAQ.md]: Should be good now.

**Comment by jawshooah** [comment]
:+1:

**Comment by fanquake** [comment]
I'm not sure this needs 5 separate commits for such simple changes, but ACK either way.

**Comment by vitorgalvao** [comment]
Merged as e9a18e907ff7fac2e968f6d264058133b1cf7d9b.

**Pull request closed without merging**